charts/ziti-router/values.yaml

```diff
@@ -151,6 +151,10 @@ image:
   # template).
   # -- deployment container command args and opts
   args: [ 'run', '{{ .Values.configMountDir }}/{{ .Values.configFile }}']
+  # -- additional arguments can be passed directly to the container to modify ziti runtime arguments
+  additionalArgs: []
+  # -- additional environment variables to pass into runtime
+  env: []
```

Comment:
No need to provide the default value for router's env again because the empty dict is defined at top level on L37. You can drop L157 and it's doc comment.